src/ripple/overlay/impl/PeerImp.cpp

```diff
@@ -1843,13 +1844,14 @@ PeerImp::getLedger (std::shared_ptr<protocol::TMGetLedger> const& m)
     if (packet.itype () == protocol::liTS_CANDIDATE)
     {
         // Request is for a transaction candidate set
-        p_journal_.trace <<
+        if (p_journal_.trace) p_journal_.trace <<
             "GetLedger: Tx candidate set";
```

Comment:
I can't remember where, but I remember seeing the exact opposite argument on the basis that the overhead of the check is larger than the overhead of just trying to send the message. I don't have an opinion either way.